app/controllers/scenarios_controller.rb

```diff
@@ -95,6 +95,20 @@ def update
     end
   end
 
+  def enable_or_disable_all_agents
+    @scenario = current_user.scenarios.find(params[:id])
+
+    respond_to do |format|
+      if @scenario.agents.update_all disabled: params[:scenario][:disabled]
```

Comment:
I tried reloading the scenario and the agents disabled status is still false. 

![screen shot 2016-05-18 at 12 06 10 pm](https://cloud.githubusercontent.com/assets/1547934/15366204/35308178-1cf1-11e6-97a7-61f3c04e6c87.png)

The params is the same (with true status)

` Parameters: {"utf8"=>"✓", "authenticity_token"=>"3t0nIpcahAGCZq6zueMZMnrc/fUATkEqRleoT6zB96lNZF9cIjzS3hT3fZ7tZPjmlW+DVv4JhSQChlme80DAsA==", "scenario"=>{"disabled"=>"true"}, "commit"=>"Yes", "id"=>"18"}`

and no the agents did not showed disabled in the UI. 

`@scenario.agents(true)` and `@scenario.agents` bring up records but is not updated.